app/players/elves/elf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Elf(Player):
+    def __init__(self, nickname: str, musical_instrument: str) -> None:
+        super().__init__(nickname)
+        self.musical_instrument = musical_instrument

```

Comment:
The `musical_instrument` attribute should be a protected attribute. Consider renaming it to `_musical_instrument` to follow encapsulation best practices.